### ISSUE TITLE
iOS PostProcess problem when generating the Xcode project in one PC and moving to another to Build it (with fix)

### ISSUE THREAD

**Issue by dantasulisses** [AppLovin/AppLovin-MAX-Unity-Plugin#119]
Hi, there is currently a issue in the way the MaxPostProcessBuildiOS.cs script works, if you are generating the Xcode project in one PC (Unity build) and then taking this generated Xcode project to build in another PC. (this is a common setup for people doing CI/CD, which uses Linux for Unity Build, and then move to Mac to build the generated Xcode project)

At line 268, this in being made:
```
var swiftFileGuid = project.AddFile(swiftFilePath, swiftFileRelativePath, PBXSourceTree.Source);
```
This will cause the "Classes/MAXSwiftSupport.swift" file point to an Absolut Directory path, which when moving between different PCs make the build not work, because it will not be able to find the reference to this file.

By changing this line to
```
var swiftFileGuid = project.AddFile(swiftFileRelativePath, swiftFileRelativePath, PBXSourceTree.Source);
```
It will correctly work, as the reference to the file in the Xcode project is now set to a fully relative path (first parameter)

If there is no other impact that you see with this change, it would be nice to be set this by default in your unity plugin

**Comment by santoshbagadi** [comment]
Hi @dantasulisses , thank you for reporting the issue! We have tested the proposed solution and it is working as expected. We'll include the fix in our next release.

**Comment by santoshbagadi** [comment]
Hi @dantasulisses , we have released a new version of our plugin ([v5.1.0](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases/tag/release_5_1_0)) that fixes this issue.
